app/travel_processor/webapp/test/integration/Opa.qunit.html

```diff
@@ -16,6 +16,7 @@
       }'
       data-sap-ui-theme="sap_horizon"
       data-sap-ui-async="true"
+      data-sap-ui-libs="sap.m,sap.ushell,sap.fe.core"
```

Comment:
I would just do a `data-sap-ui-libs="sap.m,sap.fe.core"` (no `sap.ushell`), and maybe add the same in the index.html file